### PR TITLE
feat(search): add persons to cmd-k search

### PR DESCRIPTION
**Pull request by thmsobrmlr**
## Problem

Most of the searches in the existing UI are for persons, but they are noteably missing from the CMD-K search, as they are fetched from CH, not PG.

## Changes

This PR adds persons to the result set frontend side, by ranking them with [`k / (k + a)`](https://www.wolframalpha.com/input?i=k+%2F+%28k+%2B+a%29). Hereby k is the length of the query and a is a constant.

## How did you test this code?

Clicking around

### CODE REVIEW

File: frontend/src/lib/components/CommandBar/commandBarLogic.ts [hunk 1]
```diff
@@ -19,13 +19,9 @@ export const commandBarLogic = kea<commandBarLogicType>([
...
                 toggleSearchBar: (previousState) =>
-                    [BarStatus.HIDDEN, BarStatus.SHOW_SHORTCUTS].includes(previousState)
-                        ? BarStatus.SHOW_SEARCH
-                        : BarStatus.HIDDEN,
+                    previousState === BarStatus.SHOW_SEARCH ? BarStatus.HIDDEN : BarStatus.SHOW_SEARCH,
                 toggleActionsBar: (previousState) =>
-                    [BarStatus.HIDDEN, BarStatus.SHOW_SHORTCUTS].includes(previousState)
-                        ? BarStatus.SHOW_ACTIONS
-                        : BarStatus.HIDDEN,
+                    previousState === BarStatus.SHOW_ACTIONS ? BarStatus.HIDDEN : BarStatus.SHOW_ACTIONS,
```
Comment (thmsobrmlr) [inline frontend/src/lib/components/CommandBar/commandBarLogic.ts:24]: This fixes an issue with toggling between search and actions, see https://posthog.slack.com/archives/C064U4XCPHQ/p1701125167365489.

File: frontend/src/lib/components/CommandBar/searchBarLogic.ts [hunk 2]
```diff
@@ -1,23 +1,40 @@
...
 import { commandBarLogic } from './commandBarLogic'
+import { Tab } from './constants'
 import type { searchBarLogicType } from './searchBarLogicType'
-import { BarStatus, ResultTypeWithAll, SearchResponse, SearchResult } from './types'
+import { BarStatus, PersonResult, SearchResult } from './types'
+
+const DEBOUNCE_MS = 300
+
+function rankPersons(persons: PersonType[], query: string): PersonResult[] {
+    // We know each person matches the query. To rank them
```
Comment (benjackwhite) [inline frontend/src/lib/components/CommandBar/searchBarLogic.ts:17]: Good use of a comment 😉

**Review by benjackwhite — APPROVED** [review]
Tried it out locally and it seemed to work well (hard to say for sure with local data but didn't run into anything obviously off).

Like the solution a lot - very clean 🙌

File: frontend/src/lib/components/CommandBar/searchBarLogic.ts (hunk 2 above)
Comment (thmsobrmlr) [reply frontend/src/lib/components/CommandBar/searchBarLogic.ts:17]: Do you mean it's superflous?

**Pull request merged**